doc/howto/sms_reminder_experiments.md

```diff
@@ -0,0 +1,78 @@
+# Running SMS reminder experiments
+
+## Prerequisites
+To start an SMS experiment you'll need to gather the following information:
+
+- A `name` for the experiment.
+- The `type` of experiment - it can target either "Current" patients or "Stale" patients.
+- The number of `max_patients_per_day` that will be enrolled per day.
+- `start_time` - enrollments will begin at this time.
+- `end_time` - enrollments will stop at this time .
+- `treatment_groups` - Each treatment group will have a different content or frequency of messages sent to it.
+    - You will probably also need a `control` group as a baseline in addition to all the groups that will receive messages.
+- The content and frequency of messages in each treatment group. This will be captured in the
+  `reminder_templates` of each group.
+  For current patients, the `remind_on` of reminder templates is set relative to the date
+  they are expected to return. For stale patients, it's the date of enrollment.
+- If the experiment has new messages and translations you'll need to get them [approved by DLT](doc/howto/bsnl/sms_reminders.md) and make sure they're present in
+  [config](../config/data/bsnl_templates.yml).
+
+## Setup
+- Enable the `experiment` and `notifications` flipper flags.
+- Create a data migration in the style of [this data migration](db/data/20220412130957_create_apr2022_ihci_experiment.rb).
+It should setup the `experiment`, its `treatment_groups` and `reminder_templates`.
+- Put a guard clause to make sure the migration runs only in the env you want. Write a spec for the migration.
+
+**Note about consecutive experiments**
+
+An experiment that sends notifications in advance enrolls patients who have appointments in the future.
+For an experiment starting right after, this can mean that all the patients might've already been enrolled
+in the first one. Make sure there's at least a gap of these many days (only if this value is positive) to avoid this
+```ruby
+ experiment_1 = Experimentation::Experiment.find("old experiment's id")
+ experiment_2 = Experimentation::Experiment.find("new experiment's id")
+ experiment_1.earliest_remind_on - experiment_2.earliest_remind_on
+```
```

Comment:
I think we can be clearer about this problem, and may not need the "these many days" math.

Could we say something like:
When scheduling experiments one after the other, ensure that there is enough time between experiments for...

I actually don't exactly understand the problem either, from this description 🤔 Could you explain this to me?

Reply:
Technically, it's allowed to have experiments start right next to each other. Although, if the first experiment sends reminders _before_ their appointment it'll enroll people who have appointments "in the future". If the second experiment wants to send reminders _after_ appointments is missed, there won't be any eligible patients left to enroll because they all got enrolled by the previous experiment. This causes the second experiment to end up with a tiny sample size.

Could maybe explain this over a call and you could help me write this, a little out of words 😛

Reply:
Yes I need explanation haha. We can update offline. Ship this thing before it stale-ifies any more.